### ISSUE TITLE
GoLink: warning: package "github.com/golang/protobuf/protoc-gen-go/descriptor" is provided by more than one rule

### ISSUE THREAD

**Issue by junghoahnsc** [bazelbuild/rules_go#1552]
After I upgraded to 0.11.2, I can see many warnings like this:

```
INFO: From GoLink a/b/c/.../linux_amd64_stripped/multiassigner_test:
GoLink: warning: package "github.com/golang/protobuf/protoc-gen-go/descriptor" is provided by more than one rule:
    @com_github_golang_protobuf//protoc-gen-go/descriptor:go_default_library
    @io_bazel_rules_go//proto/wkt:descriptor_go_proto
Set "importmap" to different paths in each library.
This will be an error in the future.
```

There seem to be a few opened issues that are related to this.
Is this a known issue and will be fixed?

Thanks,

**Comment by rogerhub** [comment]
The [custom BUILD rules for com_github_golang_protobuf](https://github.com/bazelbuild/rules_go/blob/master/third_party/com_github_golang_protobuf/protoc-gen-go/descriptor/BUILD.bazel.in) suggest that the first rule is an alias of the second rule. Are you declaring your own `com_github_golang_protobuf` repository rule?

**Comment by jayconrod** [comment]
As @rogerhub pointed out, I "fixed" this in master, but that fix is causing problems with builds that include pre-generated .pb.go files but don't use `go_proto_library`, so it will be rolled back.

#1548 is the plan to deal with this once and for all.

**Comment by junghoahnsc** [comment]
Yes, we have our own:

```
go_repository(
        name = "com_github_golang_protobuf",
        importpath = "github.com/golang/protobuf",
        tag = "v1.1.0",
    )
```

**Comment by jayconrod** [comment]
The fixes for #1548 have landed, and release 0.13.0 will be tagged in the next day or two with those fixes.

Note that the rules in `com_github_golang_protobuf` have some minor customization compared to what Gazelle generates, but I think if you add `build_file_proto_mode = "disable_global"` in that rule, that will get you very close.

Please reopen if you have issues with 0.13.0.
